### PR TITLE
Add some serverless discussion

### PR DESCRIPTION
**Pull request by mattmassicotte**
I've been getting more and more into using Swift on the server, but I'm very into serverless. Thought it could be worth a mention, especially because there are a bunch of great packages out there.

### CODE REVIEW

**Comment by AndrewBarba** [comment]
@mattmassicotte Here are two packages I built that make serverless Swift a heck of a lot more practical:

- Deploy to Vercel - https://github.com/swift-cloud/Vercel
- Deploy to Fastly Compute@Edge https://github.com/swift-cloud/Compute

**Comment by marius-se** [comment]
Sounds good to me, however I think we should keep it more compact by e.g. not mentioning DynamoDB and the trade-offs/advantages. What do you think @0xTim?

Also, are there other serverless functions we support? (thinking Google Cloud Functions, Azure, ....)

**Comment by mattmassicotte** [comment]
> @mattmassicotte Here are two packages I built that make serverless Swift a heck of a lot more practical:
> 
> * Deploy to Vercel - https://github.com/swift-cloud/Vercel
> * Deploy to Fastly Compute@Edge https://github.com/swift-cloud/Compute

These look cool! I'm not super-familiar with the underlying technologies being used, but seems like it could be a great addition. Open a PR!

**Comment by mattmassicotte** [comment]
I agree the database discussion is confusing here. I just wanted to mention it because it's typically a key part of a low-cost serverless deployment. But, its much more important this be informational! Will update.

**Comment by mattmassicotte** [comment]
@marius-se tried to do as you asked. I was going to move the Cassandra driver link to the top-level, but that whole area feels so Vapor-specific that I wasn't sure that was what you'd want.

File: Content/index.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ There are some [awesome projects](https://github.com/swiftwasm/awesome-swiftwasm
 
 For more information about Swift and WebAssembly, check out the [Swift and WebAssembly Book](https://book.swiftwasm.org).
 
+### Serverless?
+
+Running and managing your own servers can be complex! Cloud providers offer managed services that take care of most, if not all of the operational burden. This is particularly nice for small systems, because many of these have a free tier.
+
+- [AWS Lambda](https://github.com/swift-server/swift-aws-lambda-runtime)
+- [AWS Lambda Events](https://github.com/swift-server/swift-aws-lambda-events)
+- [Cassandra Driver](https://github.com/apple/swift-cassandra-client)
```
Comment (0xTim) [inline Content/index.md]: Let's move this alongside ElastecSearch and Redis

Comment (0xTim) [reply Content/index.md]: We can add Vercel and Fastly unless @AndrewBarba wants to add them in a separate PR

**Comment by mattmassicotte** [comment]
Ok, I believe I've incorporated the requested changes, including the additional links from @AndrewBarba

**Review by 0xTim — APPROVED** [review]

**Pull request merged**